openhtf/output/web_gui/prebuilt/scripts/app.bundle.js

```diff
@@ -41193,7 +41193,7 @@ webpackJsonp([0],[
 	    Prompt.prototype.sendResponse = function (input, id) {
 	        var ip = this.routeParams.get('ip');
 	        var port = this.routeParams.get('port');
-	        var response = input.value || '';
+	        var response = input ? input.value : '';
```

Comment:
haha guilty as charged ^.^ trying to reign in on that all around, myself included, now that we're all official-like :)